<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1414</width>
    <height>909</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Fix Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="3">
       <widget class="Line" name="line_2">
        <property name="styleSheet">
         <string notr="true">background-color: #5D5D5D</string>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="videoControls" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="videoStopBtn">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../icons/PlayerStop.png</normaloff>../icons/PlayerStop.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="videoPreviousBtn">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../icons/PlayerPrevious.png</normaloff>../icons/PlayerPrevious.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="videoPlayBtn">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../icons/PlayerPlay.png</normaloff>../icons/PlayerPlay.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="videoPauseBtn">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../icons/PlayerPause.png</normaloff>../icons/PlayerPause.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="videoNextBtn">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../icons/PlayerNext.png</normaloff>../icons/PlayerNext.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2" rowspan="2">
       <widget class="QWidget" name="sideWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>ID</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QPushButton" name="carInfoSaveBtn">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Car Information&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="textFormat">
              <enum>Qt::RichText</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="carIDEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="Line" name="line">
        <property name="styleSheet">
         <string notr="true">background-color: #5D5D5D</string>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QWidget" name="videoFrames" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QSlider" name="framesSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="frameSpinBox"/>
         </item>
         <item>
          <widget class="QLabel" name="framesCountLabel">
           <property name="text">
            <string>/ 1000</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="videoParent" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #131313</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QListWidget" name="neighborFrames">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>128</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #131313;
color: white;</string>
        </property>
        <property name="iconSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="flow">
         <enum>QListView::LeftToRight</enum>
        </property>
        <property name="isWrapping" stdset="0">
         <bool>false</bool>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1414</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar { background-color: #454545; }</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Video"/>
    <addaction name="actionOpen_CSV"/>
    <addaction name="actionSave_CSV"/>
    <addaction name="actionSave_CSV_As"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_Arc"/>
    <addaction name="actionRemove_Arc"/>
    <addaction name="actionAdd_Region"/>
    <addaction name="actionAdd_vRegion"/>
    <addaction name="actionRemove_Region"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionAdd_Index"/>
    <addaction name="actionAdd_StopVehicle"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Application"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #454545</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Arc"/>
   <addaction name="actionRemove_Arc"/>
   <addaction name="actionAdd_Region"/>
   <addaction name="actionAdd_vRegion"/>
   <addaction name="actionRemove_Region"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionAdd_Index"/>
   <addaction name="actionAdd_StopVehicle"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAdd_Arc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/ArcAdd.png</normaloff>../icons/ArcAdd.png</iconset>
   </property>
   <property name="text">
    <string>Add Arc</string>
   </property>
   <property name="toolTip">
    <string>Add Arc</string>
   </property>
  </action>
  <action name="actionRemove_Arc">
   <property name="icon">
    <iconset>
     <normaloff>../icons/ArcRemove.png</normaloff>../icons/ArcRemove.png</iconset>
   </property>
   <property name="text">
    <string>Remove Arc</string>
   </property>
   <property name="toolTip">
    <string>Remove Arc</string>
   </property>
  </action>
  <action name="actionAdd_Region">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/RegionAdd.png</normaloff>../icons/RegionAdd.png</iconset>
   </property>
   <property name="text">
    <string>Add Region</string>
   </property>
   <property name="toolTip">
    <string>Add Region</string>
   </property>
  </action>
  <action name="actionAdd_vRegion">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/RegionAdd.png</normaloff>../icons/RegionAdd.png</iconset>
   </property>
   <property name="text">
    <string>Add vRegion</string>
   </property>
   <property name="toolTip">
    <string>Add vRegion</string>
   </property>
  </action>
  <action name="actionAdd_Index">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/RegionAdd.png</normaloff>../icons/RegionAdd.png</iconset>
   </property>
   <property name="text">
    <string>Add Index</string>
   </property>
   <property name="toolTip">
    <string>Add vRegion</string>
   </property>
  </action>
  <action name="actionRemove_Region">
   <property name="icon">
    <iconset>
     <normaloff>../icons/RegionRemove.png</normaloff>../icons/RegionRemove.png</iconset>
   </property>
   <property name="text">
    <string>Remove Region</string>
   </property>
   <property name="toolTip">
    <string>Remove Region</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset>
     <normaloff>../icons/ZoomIn.png</normaloff>../icons/ZoomIn.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset>
     <normaloff>../icons/ZoomOut.png</normaloff>../icons/ZoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionAdd_StopVehicle">
   <property name="icon">
    <iconset>
     <normaloff>../icons/ZoomOut.png</normaloff>../icons/ZoomOut.png</iconset>
   </property>
   <property name="text">
    <string>ADD StopVehicle</string>
   </property>
  </action>
  <action name="actionAbout_Application">
   <property name="text">
    <string>About Application</string>
   </property>
  </action>
  <action name="actionOpen_Video">
   <property name="text">
    <string>Open Video</string>
   </property>
  </action>
  <action name="actionSave_CSV">
   <property name="text">
    <string>Save CSV</string>
   </property>
  </action>
  <action name="actionOpen_CSV">
   <property name="text">
    <string>Open CSV</string>
   </property>
  </action>
  <action name="actionSave_CSV_As">
   <property name="text">
    <string>Save CSV As...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
